.ci/oci-dashboard-puppeteer.sh

```diff
@@ -33,12 +33,11 @@ source <(curl -s ${DEVWORKSPACE_HAPPY_PATH}/common.sh)
 setupCheDashboard() {
   # Deploy Eclipse Che with a custom dashboard image
   cat >/tmp/che-cr-patch.yaml <<EOF
+apiVersion: org.eclipse.che/v2
 spec:
   components:
     dashboard:
-      deployment:
-        containers:
-          - image: ${CI_CHE_DASHBOARD_IMAGE}
+      dashboardImage: '${CI_CHE_DASHBOARD_IMAGE}'
```

Comment:
```suggestion
      deployment:
        containers:
          - image: ${CI_CHE_DASHBOARD_IMAGE}
             imagePullPolicy: Always
```

Reply:
@ScrewTSW As I remember, ```dashboardImage: ...``` should refer to ```apiVersion: org.eclipse.che/v1```. 
WDYT?

Reply:
Good point. let me fix that in the follow-up PR